src/training/ModelFiller.cpp

```diff
@@ -1,5 +1,12 @@
 #include <ecto/ecto.hpp>
-#include <opencv2/core/core.hpp>
+
+
+#if CV_VERSION_MAJOR < 3
```

Comment:
this is now the only proper way of checking for OpenCV2 / 3: https://github.com/wg-perception/opencv_candidate/blob/master/src/opencv_candidate/feature2d.cpp#L113